Logger/Logger.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Logger.LoggerDestinations;
+
+namespace Logger
+{
+    public class SimpleLogger : ILogger
+    {
+        private readonly ILoggerDestination destination;
+
+        public SimpleLogger(Destinations destinations)
+        {
+            destination = destinations switch
+            {
+                Destinations.Console => new ConsoleLogger(),
+                _ => throw new ArgumentOutOfRangeException(nameof(destinations), destinations, null)
+            };
+        }
+
+        public void Error(string message)
```

Comment:
Don't you think all methods above are similar? We can simplify the logic
```
        public void Error(string message)
        {
            Write(LogLevel.Error, message);
        }

        public void Error(Exception ex)
        {
            Write(LogLevel.Error, $"Type: {ex.GetType()} Message: {ex.Message}");
        }

        public void Info(string message)
        {
            Write(LogLevel.Info, message);
        }

        public void Warning(string message)
        {
            Write(LogLevel.Warning, message);
        }

        private void Log(LogLevel level, string message)
        {

        }
 ```

Reply:
Fixed